@here/olp-sdk-dataservice-read/lib/client/StreamLayerClient.ts

```diff
@@ -23,9 +23,14 @@ import {
     DataStoreRequestBuilder,
     HRN,
     OlpClientSettings,
-    RequestFactory
+    PollRequest,
+    RequestFactory,
+    SubscribeRequest
 } from "..";
-import { SubscribeRequest } from "./SubscribeRequest";
+
+export interface SubscribeResponse {
```

Comment:
Please add a comment about that interface and why it can be useful for user

Reply:
It is redundant interface. Removed